GameData/KCS/Patches/Scanners.cfg

```
@@ -0,0 +1,29 @@
+
+@PART[SurveyScanner]
+{
+   	%MODULE[ModuleObjectTracking]
+  	{
+		moduleName = ModuleObjectTracking
+		baseDetectionRange = 5000
+	}
+}
+
+@PART[OrbitalScanner]
+{
+   	MODULE
+  	{
+		name = ModuleObjectTracking
+		baseDetectionRange = 1000
+  	}
+}
+
+//give all AI modules an inherit visual range
```

Clarification about what object tracking does at the moment.

```suggestion
// Give all AI modules an inherent lock range.
```